vllm/model_executor/layers/fused_moe/cutlass_moe.py

```diff
@@ -460,7 +460,7 @@ def cutlass_moe_fp8(
     expert_map: torch.Tensor | None = None,
     apply_router_weight_on_input: bool = False,
     global_num_experts: int = -1,
-    parallel_config=None,
+    moe_parallel_config=None,
```

Comment:
I don't think these extra parameters are necessary.  This version of the cutlass kernels is never used in an EP context.

Reply:
This function is specifically intended for the non-EP case, i.e. it uses `MoEPrepareAndFinalizeNoEP` for constructing the modular kernel.  Similarly with other use sites.

Reply:
Fixed